### PR TITLE
Fix Suspicious Login Attempt - Wrong login data

### PR DESCRIPTION
**Pull request by nilkesede**
I put together #1079 #1064 in order to solve the `suspicious attempt` problem.
I'm not a python programmer so feel free to correct me if I made some mistake.

### CODE REVIEW

**Comment by Naramsim** [comment]
Could you please change `pickle` with `json`? Reasons:

* Having cookies saved as `json` is much more readable (human-readable). 

* With `json` we can dump/update by ourselves the cookie file if we are running headless.

* `Json` and pickle have basically the same methods (`dumps`, `loads`).

* We don't really need to save cookies as a binary file.

* `Json` is much [faster](https://konstantin.blog/2010/pickle-vs-json-which-is-faster/) and [safer](https://stackoverflow.com/a/2259351/3482533)

File: instapy/instapy.py [hunk 1]
```diff
@@ -132,6 +132,10 @@ def __init__(self,
                          'We\'re working to fix this soon!')
             self.logger.critical(error_msg)
 
+    def get_dont_like(self):
```
Comment (converge) [inline instapy/instapy.py]: do we need it ?

**Review by converge — CHANGES_REQUESTED** [review]
hey mate! nice ideas!

Unfortunatelly it doesn't login for me on my VPS, I receive the wrong login data message.

another point is, we should avoid try to bypass suspicious attempt page every time we login, it increases the delay to login.

File: instapy/instapy.py (hunk 1 above)
Comment (nilkesede) [reply instapy/instapy.py]: we don't really need it

**Comment by nilkesede** [comment]
thanks, man!
it doesn't try to bypass every time, just when we have the suspicious attempt page. but i got you, it really increases the delay to login.

**Comment by converge** [comment]
#1074 #1125

**Pull request closed without merging**